draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -13001,6 +13001,11 @@ transfer-coding = &lt;transfer-coding, see <xref target="Messaging" x:fmt="," x:
    values are present was clarified.
    (<xref target="field.vary"/>)
 </t>
+<t>
+   Parameters in media type, media range, and expectation can be empty via
+   one or more trailing semicolons.
+   (<xref target="parameter"/>)
+</t>
```

Comment:
Hm.

If I have 

    text/plain ; foo=bar ; ; charset=qux ;

are all of these that we now allow "trailing"?

Reply:
...also this change seems to apply to a different issue...